### PR TITLE
added some features

### PR DESCRIPTION
**Pull request by danielme85**
Hi,
we needed to get users, collections and collection assets so I added that. 
Might be useful for others...?

### CODE REVIEW

**Comment by danielsequeira** [comment]
@danielme85 thanks for the pull request! Could you also update the README with the new methods that you added?

File: src/Bynder/Api/Impl/AssetBankManager.php [hunk 1]
```diff
@@ -391,4 +391,36 @@ public function deleteUsage($query)
             ['query' => $query]
         );
     }
+
+    /**
+     * Gets all collections based on query
+     *
+     * @param null|array $query Query parameters for the Bynder API.
+     *
+     * @return \GuzzleHttp\Promise\Promise Response of asset usage delete.
```
Comment (condemil) [inline src/Bynder/Api/Impl/AssetBankManager.php]: The description is not relevant.

File: src/Bynder/Api/Impl/AssetBankManager.php [hunk 2]
```diff
@@ -391,4 +391,36 @@ public function deleteUsage($query)
...
+            ['query' => $query]
+        );
+    }
+
+    /**
+     * Gets all media assets related to a collection
+     *
+     * @param string $collectionId The collection id from Bynder.
+     *
+     * @return \GuzzleHttp\Promise\Promise Response of asset usage delete.
```
Comment (condemil) [inline src/Bynder/Api/Impl/AssetBankManager.php]: The description is not relevant.

File: src/Bynder/Api/IAssetBankManager.php [hunk 3]
```diff
@@ -254,4 +254,28 @@ public function getUsage($query);
      * @throws \GuzzleHttp\Exception\RequestException When request fails.
      */
     public function deleteUsage($query);
+
+    /**
+     * Gets all collections based on query
+     *
+     * @param $query
+     *
+     * @return \GuzzleHttp\Promise\Promise Response of asset usage delete.
```
Comment (condemil) [inline src/Bynder/Api/IAssetBankManager.php]: The description is not relevant.

File: src/Bynder/Api/IAssetBankManager.php [hunk 4]
```diff
@@ -254,4 +254,28 @@ public function getUsage($query);
...
+     * @throws \Exception
+     */
+    public function getCollections($query);
+
+    /**
+     * Gets all media assets related to a collection
+     *
+     * @param string $collectionId The collection id from bynder.
+     *
+     * @return \GuzzleHttp\Promise\Promise Response of asset usage delete.
```
Comment (condemil) [inline src/Bynder/Api/IAssetBankManager.php]: The description is not relevant.

**Comment by danielme85** [comment]
> @condemil we have the same comments to describe methods throughout the entire SDK code, so I would actually keep them for consistency purposes

I did leave in some irrelevant stuff from my reckless copy-pasting:  
"Response of asset usage delete." <- dont really apply to the new methods. 
I assumed that was the culprit here :)

**Pull request merged**